src/acpi_types.rs

```diff
@@ -86,10 +86,16 @@ impl MemoryAffinity {
         self.base_address + self.length
     }
 
+    /// Check if the region is nvm or not.
     pub fn is_non_volatile(&self) -> bool {
         self.non_volatile
     }
 
+    /// Check if the region is reserved for hotplugged memory.
+    pub fn is_hotplug_region(&self) -> bool {
+        self.enabled & self.hotplug_capable
```

Comment:
Is this supposed to be a bitwise and or missing a `&`?

Reply:
Typo. Fixed!